weblate/trans/views/js.py

```diff
@@ -151,7 +151,7 @@ def ignore_check_source(request, check_id):
         unit.save(same_content=True)
 
     # response for AJAX
-    return HttpResponse("ok")
+    return HttpResponse(unit.extra_flags)
```

Comment:
In case we do client side processing, it should be a JSON response here. Also, it will need more content:

* `unit.extra_flags` should be stored into popup editor for flags
* `unit.all_flags.format()` should be used as display value

Reply:
This is now a JSON response. I included "unit.extra_flags", "unit.all_flags", and the single "ignore" flag to the response. I think it might be nice to also have a notification letting the user know that the single flag was added to the list of extra flags. I'm not sure if there is a popup notification system in weblate that can be hooked into?